Source/santad/SNTExecutionController.mm

```diff
@@ -538,4 +603,17 @@ - (void)createRuleForStandaloneModeEvent:(SNTStoredEvent *)se {
   // TODO: Notify the sync service of the new rule.
 }
 
+extern "C" int pid_suspend(pid_t pid);
+extern "C" int pid_resume(pid_t pid);
+
+// Wrapper around the pid_suspend() / pid_resume() / kill() functions that uses signal numbers
+// to determine which to use and which can be easily mocked out in tests;
+- (void)sendSignal:(int)signal toPID:(pid_t)pid {
```

Comment:
Conceptually I'm not a fan of this naming since 2 of 3 operations aren't a signal. Maybe add some new enum type:
```
enum class ProcessControl {
  kSuspend,
  kResume,
  kKill,
};
```
and change the signature to something like
```
- (void)manipulateProcess:(pid_t)pid withControl:(ProcessControl)control
```